### PR TITLE
Edits and updates to First Steps and Intro docs

### PR DESCRIPTION
**Pull request by charlesthompson3**
# Description of change

Edited and updated content for both First Steps and Introduction documentation for Getting Started; added new section "Chrysalis" as an overview within Introduction. Ready for SME review.

# Type of change

- Enhancement (a non-breaking change which adds new content)

# Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my changes

### CODE REVIEW

File: getting-started/1.5/first-steps/hello-world.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ In this step, you create a zero-value transaction that contains a "Hello world"
     ```
 
     :::info:
-    This address does not have to belong to anyone. To be valid, the address just needs to consist of 81 trytes. You'll learn about trytes in [Ternary](../the-tangle/ternary.md). For now, it's enough to know that a tryte is just a character between A-Z or the number 9.
+    This address does not have to belong to anyone. To be valid, the address just needs to consist of 81 trytes. You'll learn about trytes in bech32. For now, it's enough to know that a tryte is just a character between A-Z or the number 9.
```
Comment (fijter) [inline getting-started/1.5/first-steps/hello-world.md:45]: Bech32 format is not in trytes and not 81 trytes long, it's a totally different format. A mainnet address looks like `iota1q9f0mlq8yxpx2nck8a0slxnzr4ef2ek8f5gqxlzd0wasgp73utryj0w6qwt` and a testnet one has the `atoi` prefix: `atoi1q9f0mlq8yxpx2nck8a0slxnzr4ef2ek8f5gqxlzd0wasgp73utryjgqtp5x`. See https://github.com/Wollac/protocol-rfcs/blob/bech32-address-format/text/0020-bech32-address-format/0020-bech32-address-format.md

File: getting-started/1.5/introduction/about-iota.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Everything in the Tangle is immutable and transparent. Through cryptography, you
 
 ### Anyone who values security
 
-IOTA uses quantum-robust signatures to protect the network against the next generation of computing power.
+IOTA uses multiple, secure signatures to prorect the network agaunst the next generation of computing power.
```
Comment (fijter) [inline getting-started/1.5/introduction/about-iota.md:32]: `prorect`, `agaunst`

**Review by fijter — COMMENTED** [review]
Hey Charles, quickly ran through it and found some things to change.

File: getting-started/1.5/introduction/about-iota.md (hunk 2 above)
Comment (charlesthompson3) [inline getting-started/1.5/introduction/about-iota.md:32]:
```suggestion
IOTA uses multiple, secure signatures to protect the network against the next generation of computing power.
```

**Comment by charlesthompson3** [comment]
@fijter, updated content mentioned in your comments

**Pull request merged**